examples/all/manifests/thanos-rule-statefulSet.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         - --objstore.config=$(OBJSTORE_CONFIG)
         - --data-dir=/var/thanos/rule
         - --label=rule_replica="$(NAME)"
```

Comment:
Assuming this should create the same sort of problem for this as well, no?

Reply:
No, this syntax is good for add label.